proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -20,12 +20,15 @@ To solve this problem, add `bluetoothDeviceAddress` and `requiresAudioSupport` s
 The current SDL Java Suite library cancels the transport connection if the `requiresAudioSupport` setting is TRUE and BT A2DP is not connected.
 However, with this proposal, by adding a new parameter, the SDL app always establishes the transport connection and is registered without depending on the connection status of BT A2DP. If BT A2DP is not connected, the HU will automatically connect BT using `bluetoothDeviceAddress` or request connection from the user.
 
+Note: In the case of other platform (such as iOS and JavaScript Suite), the system can output the audio via USB. Therefore, other platform can omit the requiresAudioSupport parameter.
+
 ### Change the app registration flow
 
 In order for the app to be always registered, the changes in SDL session establishment to app registration flow are shown below.
+This flow is only for Android devices.
 If the SDL app or HU cannot use this function, it will perform the same operation as before.
 
-![registeringApp_flow](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow1.png)
+![registeringApp_flow](../assets/proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress/registeringApp_flow.png)
```

Comment:
In the top right corner of the image, you've added the below. Can you please advise which requested revision (from [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/941#issuecomment-675669406)) required this change? 

<img width="929" alt="Screen Shot 2020-09-08 at 1 27 02 PM" src="https://user-images.githubusercontent.com/22600496/92508671-01e0e500-f1d7-11ea-8923-7f4e3e7ae4b1.png">

Reply:
@theresalech -san,
This change is referred below.
I think this change is referred "Update to use flow chart".
What you say is to add OnDeviceStateChanged to the HMI API. And it is not need to add the flow chart.
Is it correct?
If yes, I will modify to delete new part from the flow chart.

>  Update to use flow chart in this comment. Also, add BluetoothInfo struct to OnDeviceStateChanged, with bluetoothDeviceAddress and a2dpConnectionState params as described in this comment,

Reply:
@Akihiro-Miyazaki, I see. The chart is okay as is then. Thanks!